<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QStackedWidget" name="login_SW">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>226</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>비밀번호 찾기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signup_Button">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>340</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GyeonggiBatang</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 15pt &quot;GyeonggiBatang&quot;;
	color: rgb(251, 86, 7);
	background-color: white;
	border: 2px solid rgb(251, 86, 7);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_4">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>224</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>아이디 찾기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_Button">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>280</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GyeonggiBatang</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 15pt &quot;GyeonggiBatang&quot;;
	color: white;
	background-color: rgb(251, 86, 7);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_id_lineEdit">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>110</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_pw_lineEdit">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>170</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>185</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>비밀번호</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_2">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>125</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;경기천년바탕 Regular&quot;;</string>
     </property>
     <property name="text">
      <string>아이디</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>35</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;경기천년바탕 Regular&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;로그인&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Home1_Button">
     <property name="geometry">
      <rect>
       <x>475</x>
       <y>40</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 13pt &quot;GyeonggiBatang&quot;;
	color: white;
	background-color: rgb(251, 86, 7);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
    <zorder>signup_Button</zorder>
    <zorder>sh_label_4</zorder>
    <zorder>login_Button</zorder>
    <zorder>login_id_lineEdit</zorder>
    <zorder>login_pw_lineEdit</zorder>
    <zorder>sh_label_3</zorder>
    <zorder>sh_label_2</zorder>
    <zorder>sh_label</zorder>
    <zorder>Home1_Button</zorder>
    <zorder>label_5</zorder>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="sh_label_7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>8</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;GyeonggiBatang&quot;;
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;회원가입&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_8">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>148</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;아이디&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_9">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>198</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;비밀번호&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_10">
     <property name="geometry">
      <rect>
       <x>63</x>
       <y>248</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;비밀번호 확인&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_11">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>301</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;연락처&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_12">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>348</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;주소&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_13">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>98</y>
       <width>65</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;이름&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_lineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>138</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="pw_lineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>188</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="pw2_lineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>238</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="phone_lineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>288</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>숫자만 입력하세요</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="address_lineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>338</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_lineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>78</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="duplication_Button">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>140</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 11pt &quot;GyeonggiBatang&quot;;
	color: white;
	background-color: rgb(251, 86, 7);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>중복확인</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_14">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>399</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GyeonggiBatang</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;비밀번호 찾기 질문&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_15">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>450</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GyeonggiBatang</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;답변&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="answer_lineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>440</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="Question">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>390</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>기억에 남는 추억의 장소는?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>자신의 인생 좌우명은?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>자신의 보물 제 1호는?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>가장 기억에 남는 선생님 이름은?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>추억하고 싶은 날짜가 있다면?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>내가 좋아하는 캐릭터는?</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="Home2_Button">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>530</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 12pt &quot;GyeonggiBatang&quot;;
	color: rgb(251, 86, 7);
	background-color: white;
	border: 2px solid rgb(251, 86, 7);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>로그인 페이지로</string>
     </property>
    </widget>
    <widget class="QPushButton" name="join_Button">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>530</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 12pt &quot;GyeonggiBatang&quot;;
	color: white;
	background-color: rgb(251, 86, 7);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>가입하기</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLineEdit" name="name_lineEdit_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_16">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>171</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;연락처&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_17">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>65</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;이름&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="phone_lineEdit_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>158</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>숫자만 입력하세요</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_18">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;GyeonggiBatang&quot;;
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;아이디 찾기&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Home2_Button_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>240</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 12pt &quot;GyeonggiBatang&quot;;
	color: rgb(251, 86, 7);
	background-color: white;
	border: 2px solid rgb(251, 86, 7);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>찾기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Home2_Button_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>480</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 12pt &quot;GyeonggiBatang&quot;;
	color: rgb(251, 86, 7);
	background-color: white;
	border: 2px solid rgb(251, 86, 7);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>로그인창으로</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>320</y>
       <width>371</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디 표시 Label</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="sh_label_19">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;GyeonggiBatang&quot;;
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;비밀번호 찾기&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_lineEdit_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Home2_Button_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>490</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 12pt &quot;GyeonggiBatang&quot;;
	color: rgb(251, 86, 7);
	background-color: white;
	border: 2px solid rgb(251, 86, 7);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>로그인창으로</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>330</y>
       <width>371</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>비밀번호 표시 label</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_21">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>65</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;아이디&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Question_2">
     <property name="geometry">
      <rect>
       <x>203</x>
       <y>144</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>기억에 남는 추억의 장소는?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>자신의 인생 좌우명은?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>자신의 보물 제 1호는?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>가장 기억에 남는 선생님 이름은?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>추억하고 싶은 날짜가 있다면?</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>내가 좋아하는 캐릭터는?</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="sh_label_22">
     <property name="geometry">
      <rect>
       <x>37</x>
       <y>153</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GyeonggiBatang</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;비밀번호 찾기 질문&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="answer_lineEdit_2">
     <property name="geometry">
      <rect>
       <x>203</x>
       <y>194</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="sh_label_23">
     <property name="geometry">
      <rect>
       <x>93</x>
       <y>204</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GyeonggiBatang</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;GyeonggiBatang&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;답변&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Home2_Button_5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>260</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font: 12pt &quot;GyeonggiBatang&quot;;
	color: rgb(251, 86, 7);
	background-color: white;
	border: 2px solid rgb(251, 86, 7);
	border-radius: 5px;
}</string>
     </property>
     <property name="text">
      <string>찾기</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>login_id_lineEdit</tabstop>
  <tabstop>login_pw_lineEdit</tabstop>
  <tabstop>login_Button</tabstop>
  <tabstop>name_lineEdit</tabstop>
  <tabstop>id_lineEdit</tabstop>
  <tabstop>pw_lineEdit</tabstop>
  <tabstop>pw2_lineEdit</tabstop>
  <tabstop>phone_lineEdit</tabstop>
  <tabstop>address_lineEdit</tabstop>
  <tabstop>Question</tabstop>
  <tabstop>answer_lineEdit</tabstop>
  <tabstop>Home2_Button</tabstop>
  <tabstop>join_Button</tabstop>
  <tabstop>duplication_Button</tabstop>
  <tabstop>signup_Button</tabstop>
  <tabstop>Home1_Button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
